<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>半径：</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="radiusDoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1231321231346546.000000000000000</double>
   </property>
   <property name="value">
    <double>3.980000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>面积：</string>
   </property>
  </widget>
  <widget class="QLabel" name="areaLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="calculateBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>归零</string>
   </property>
  </widget>
  <widget class="QFontComboBox" name="fontComboBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="clearContentBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>470</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>清空结果</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearAllBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
